### DISCUSSION TITLE
Running patchcore raises ConfigKeyError

### DISCUSSION THREAD

**Discussion by ghost** [open-edge-platform/anomalib#1497] [Q&A]
I am doing the following on my custom data:
`python tools/train.py --model patchcore --config path/to/the/modified/config/file.`
When I run it, I get
```
raise ConfigKeyError(f"Missing key {key!s}")
omegaconf.errors.ConfigAttributeError: Missing key normal_test_dir
    full_key: dataset.normal_test_dir
    object_type=dict
```

The changes I made to the config file are attached below:
![image](https://github.com/openvinotoolkit/anomalib/assets/63196905/47585710-a916-4472-a9b6-6341687ead13)
![image](https://github.com/openvinotoolkit/anomalib/assets/63196905/1bd4bbc1-3bbf-4388-8511-a4acb69ab179)

When I use only
`python tools/train.py --model patchcore`,
it works fine. What is wrong with my custom case?

**Comment by blaz-r** [comment]
Hello, refer to following page in docs for correct format:
https://anomalib.readthedocs.io/en/latest/how_to_guides/train_custom_data.html
As the issue says, you will need to add `normal_test_dir`, if you don't have normal samples in test set set it to `null`
